### ISSUE TITLE
frontend: bug in voting-details-admin dialog

### ISSUE THREAD

**Issue by gilgda** [Werdenfels-Gymnasium/voting-calculator#82]
Is this meant to show like this? If so the label of the sign should be optimized.
![voting-bug](https://user-images.githubusercontent.com/16957319/33234785-b7e1516e-d22c-11e7-8d2b-489e90440a9e.JPG)

**Comment by Lordnoname** [comment]
Yeah, in the moment this is displayed properly....but i agree changing that :)

**Comment by Lordnoname** [comment]
@gilgda It would be great if you could submit a pr :)
